react/react-training/src/index.js

```diff
@@ -1,10 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import './scss/index.scss';
-import registerServiceWorker from './registerServiceWorker';
+import Game from './screens/Game';
```

Comment:
Change this for the alias with babel

Reply:
I don't understand what do you mean here.

Reply:
Remember the explication that i told you